docs/CppInteropability/GettingStartedWithC++Interop.md

```diff
@@ -0,0 +1,207 @@
+#  Getting started with C++ Interoperability
+
+This document is desgined to get you started with bidirectional API-level interoperability between Swift and C++.
+
+## Table of Contents
+
+- [Creating a Module to contain your C++ source code](#creating-a-module-to-contain-your-c-source-code)
+- [Adding C++ to an Xcode project](#adding-c-to-an-xcode-project)
+- [Creating a Swift Package](#Creating-a-Swift-Package)
+- [Building with CMake](#building-with-cmake)
+
+## Creating a Module to contain your C++ source code
+
+- Create a new C++ implementation and header file
+- For this example we will call the files Cxx, so we should have a Cxx.cpp and Cxx.hpp.
+- Next create an empty file and call it `module.modulemap`, in this file create the module for your source code, and define your C++ header (`requires cplusplus` isn't required but it's convention for C++ modules, especially if they use C++ features).
+```
+//In module.modulemap
+module Cxx {
+    //note that your header should be the file that containts your method implementations
+    header "Cxx.cpp
```

Comment:
Nit: `"Cxx.cpp"`

Reply:
Also, this should be the header file: `Cxx.hpp`.

Reply:
Ah yep, thank you!

Reply:
Sorry, I should have clarified. My original comment was about the quotation mark. There needs to be a closing quote.

Reply:
I really appreciate all the support! Stoked that i can contribute

Reply:
For sure. Thank you for contributing!